### PR TITLE
Remove unused Stockwatcher stuff, update compiled project, fix links to demos

### PR DESCRIPTION
**Pull request by manolo**
This is a big commit, but almost it is removing the stockwatcher compiled demo triplicated in different folders.

The demo only need be deployed in the /gettingstarted/ but it also was in /codeclient/ and /manageevents/. At the same time, I have recompiled the demo to join permutations and remove 'googlecode' logo by 'gwt' one.

Also fixed a link to the demo which was not working.

[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/95)

### CODE REVIEW

File: src/main/site/doc/latest/tutorial/gettingstarted/StockWatcher.css [hunk 1]
```diff
@@ -51,3 +51,7 @@ body {
 .negativeChange {
   color: red;
 }
+
+.errorMessage {
+  color: red;
+}
```
Comment (jDramaix) [inline src/main/site/doc/latest/tutorial/gettingstarted/StockWatcher.css]: No new line at the end of file

**Comment by jDramaix** [comment]
! nit then LGTM

**Comment by jnehlmeier** [comment]
Also just a nit: IMHO it looks better without the 15px padding on the iframe especially once the vertical scrollbar is visible when symbols have been added to the demo.

File: src/main/site/doc/latest/tutorial/gettingstarted/StockWatcher.css (hunk 1 above)
Comment (manolo) [inline src/main/site/doc/latest/tutorial/gettingstarted/StockWatcher.css]: done

**Comment by manolo** [comment]
@jnehlmeier removed padding, it made sense with the old compiled version. Also increased width in order not to wrap text in the change column.

**Comment by jnehlmeier** [comment]
@manolo 400px seems to cause the column to sometimes break and sometimes not which makes the whole table change its height on nearly each refresh. 450px seems to work fine for me.

**Comment by manolo** [comment]
@jnehlmeier increased width.

**Pull request merged**